kymatio/scattering2d/backend/torch_skcuda_backend.py

```diff
@@ -69,13 +69,11 @@ def __call__(self, x, k):
         signal_shape = x.shape[-3:]
         x = x.view((-1,) + signal_shape)
 
-        out = x.new(size=[x.shape[0], x.shape[1] // k, x.shape[2] // k, 2])
+        out = torch.zeros((x.shape[0], x.shape[1] // k, x.shape[2] // k, 2),
```

Comment:
`torch.empty`

Reply:
What is the shape of `x`? If it ends in 2, you can also do:
`torch.empty_like(x[:, ::k, ::k])`